### PR TITLE
VG-Prereq-Component-Software

### PR DESCRIPTION
**Pull request by vanajaguru**
PR to clarify VZ components versus software ([VZ-8982](https://jira.oraclecorp.com/jira/browse/VZ-8982))

### CODE REVIEW

**Review by rosemarymarano — APPROVED** [review]

File: content/en/docs/setup/prereqs.md [hunk 1]
```diff
@@ -56,38 +56,37 @@ See the Coherence versions supported [here](https://oracle.github.io/coherence-o
...
-| Velero                       | 1.9.1        | Manages backup and restore of Kubernetes configurations and data.                        |
-| WebLogic Kubernetes Operator | 4.0.6        | Assists with deploying and managing WebLogic domains.                                    |
-| WebLogic Monitoring Exporter | 2.1.3        | Exports Prometheus-compatible metrics from WebLogic instances.                           |
+ ## Installed components
+ Verrazzano installs a curated set of open source components and software. The following table lists each component, its software (where applicable), its version, and a brief description.
+
+ | Component                    | Software               | Version      | Description                                                                              |
+ |------------------------------|------------------------|--------------|------------------------------------------------------------------------------------------|
+ | Alertmanager                 |                             | 0.24.0       | Handles alerts sent by client applications, such as the Prometheus server.               |
```
Comment (gilbode) [inline content/en/docs/setup/prereqs.md:64]: Seems sort of strange to have this "software" column that's mostly blank.

Comment (gilbode) [reply content/en/docs/setup/prereqs.md:64]: What about just making the page say "installed software" (drop the use of the word component entirely), get rid of the extra column - use the name of the software and hyperlink to the component it refers to in the Verrazzano CR doc?

Comment (vanajaguru) [reply content/en/docs/setup/prereqs.md]: Thanks. I have removed the _Software_ column, and have provided hyperlink for the software that the component refers to. Please see [2f73ad7](https://github.com/verrazzano/docs/pull/1044/commits/2f73ad724956104c04c276790cab7564bc908e05)

**Review by gilbode — APPROVED** [review]

File: content/en/docs/setup/prereqs.md (hunk 1 above)
Comment (vanajaguru) [reply content/en/docs/setup/prereqs.md]: Rosemary, can you please review the latest PR [0744861](https://github.com/verrazzano/docs/pull/1044/commits/0744861fb94ca1daa7830d0888a249311af30f52)

Comment (MelwynPaul) [reply content/en/docs/setup/prereqs.md]: I fixed the links in [0744861](https://github.com/verrazzano/docs/pull/1044/commits/0744861fb94ca1daa7830d0888a249311af30f52)

Comment (vanajaguru) [reply content/en/docs/setup/prereqs.md]: Thank you Melwyn for fixing the links.

File: content/en/docs/setup/prereqs.md [hunk 2]
```diff
@@ -56,38 +56,37 @@ See the Coherence versions supported [here](https://oracle.github.io/coherence-o
...
-| Prometheus Operator          | 0.59.1       | Provides management for Prometheus monitoring tools.                                     |
-| Prometheus Pushgateway       | 1.4.2        | Allows ephemeral and batch jobs to expose their metrics to Prometheus.                   |
-| Rancher                      | 2.7.2        | Manages multiple Kubernetes clusters.                                                    |
-| Rancher Backup Operator      | 2.1.3        | Manages backup and restore of Rancher configurations and data.                           |
-| Velero                       | 1.9.1        | Manages backup and restore of Kubernetes configurations and data.                        |
-| WebLogic Kubernetes Operator | 4.0.6        | Assists with deploying and managing WebLogic domains.                                    |
-| WebLogic Monitoring Exporter | 2.1.3        | Exports Prometheus-compatible metrics from WebLogic instances.                           |
+## Installed software
+Verrazzano installs a curated set of open source software. The following table lists each software, its  version, and a brief description.
```
Comment (rosemarymarano) [inline content/en/docs/setup/prereqs.md:60]: -> The following table lists each one, its  version, and a brief description. OR 
The following table lists the software, its  version, and a brief description.

**Review by rosemarymarano — COMMENTED** [review]
A few questions. 

- Not sure why you used relative links for the x-refs to the Reference section APIs especially because these docs are going to be moved soon.
- Second, Mike G requested, "hyperlink to the component it refers to in the Verrazzano CR doc." You linked to only five components in the VCR. Is that all there are? I think that there are many others that are components in the VCR. 

Maybe @gilbode should review this PR again to make sure that we delivered what he wants.

File: content/en/docs/setup/prereqs.md (hunk 2 above)
Comment (vanajaguru) [reply content/en/docs/setup/prereqs.md]: Thanks. Updated the statement

File: content/en/docs/setup/prereqs.md [hunk 3]
```diff
@@ -57,36 +57,36 @@ Verrazzano supports all versions of Helidon.  For more information, see [Helidon
...
-  | Prometheus Pushgateway   | 1.4.2   | Allows ephemeral and batch jobs to expose their metrics to Prometheus.   |
-  | Rancher   | 2.6.8   | Manages multiple Kubernetes clusters.                                               |
-  | Rancher Backup Operator   | 2.1.3   | Manages backup and restore of Rancher configurations and data.      |
-  | Velero   | 1.9.1   | Manages backup and restore of Kubernetes configurations and data.                    |
-  | [WebLogic Kubernetes Operator](../../reference/api/vpo-verrazzano-v1alpha1/#install.verrazzano.io/v1alpha1.WebLogicOperatorComponent)   | 4.0.6   | Assists with deploying and managing WebLogic domains.                                    |
-  | [WebLogic Monitoring Exporter](../../reference/api/vpo-verrazzano-v1alpha1/#install.verrazzano.io/v1alpha1.WebLogicOperatorComponent)   | 2.1.3   | Exports Prometheus-compatible metrics from WebLogic instances.                           |
+  | [Alertmanager]({{< relref "/docs/customize/prometheus/" >}})   | 0.24.0   | Handles alerts sent by client applications, such as the Prometheus server.    |
```
[the remark this answers is not in the document]
Comment (vanajaguru) [reply content/en/docs/setup/prereqs.md]: I have updated the x-refs and removed AlertManager. Please review [83ac660](https://github.com/verrazzano/docs/pull/1044/commits/83ac660c9ea5dd2cb00b779ade176217cd4c4fe2)

**Review by rosemarymarano — APPROVED** [review]

**Comment by vanajaguru** [comment]
Changes merged to master

**Pull request merged**